test/cli/test_cli.expected

```diff
@@ -157,3 +157,45 @@ File "foo.ml", line 5, characters 0-0:
 Error: Syntax error
 [1]
 
+conf_bad_version
+================
+args: --impl -
+ocamlformat: Error while parsing .ocamlformat:
+             For option "bad": expecting "0.12-18-gea9fdbb-dirty" but got "bad"
```

Comment:
This will always fail as it depends on the commit number, unfortunately. You can postprocess the output but we can leave this test for now I think.

Reply:
I was hoping https://github.com/ocaml-ppx/ocamlformat/pull/1133 would solve this.
I just forgot to mark this PR as "Draft".